internal/satellite/tools/generate_plugin_doc.go

```diff
@@ -0,0 +1,89 @@
+// Licensed to Apache Software Foundation (ASF) under one or more contributor
+// license agreements. See the NOTICE file distributed with
+// this work for additional information regarding copyright
+// ownership. Apache Software Foundation (ASF) licenses this file to you under
+// the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package tools
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"reflect"
+	"sort"
+	"strings"
+
+	"github.com/apache/skywalking-satellite/internal/pkg/log"
+	"github.com/apache/skywalking-satellite/internal/pkg/plugin"
+	"github.com/apache/skywalking-satellite/plugins"
+)
+
+const (
+	docDir      = "docs"
+	docPath     = docDir + "/plugin-description.md"
```

Comment:
we could use `filepath.Join` instead of concatenating them manually.

Reply:
welcome PR to fix it.

Reply:
I think you can submit a PR directly ^_^ @nic-chen

Reply:
sure, thanks for remind.

Reply:
Using a function in `const` is not allowed, unless you want to change all `const` to `var`

Reply:
@kezhenxu94 
you're right. There is little benefit from such modification.
I submitted another pr, welcome to review, thanks.

https://github.com/apache/skywalking-satellite/pull/15